pkgs/top-level/all-packages.nix

```diff
@@ -16297,6 +16297,8 @@ with pkgs;
     withGui = false;
   };
 
+  veroroute = libsForQt5.callPackage ../by-name/ve/veroroute/package.nix { };
```

Comment:
Don't introduce new entries in all-packages.nix if possible (see failing nixpkgs-vet CI)

Reply:
Thanks, done!